### PR TITLE
ADD: WP constants in .env & application.php

### PR DESCRIPTION
**Pull request by DSGND**
Add some WP constants and default value if not defined in .env file.

### CODE REVIEW

**Review by QWp6t — CHANGES_REQUESTED** [review]
Thanks for the PR!

There are some issues that would need to be updated before we'd be able to merge this.

I've added some comments. I would also encourage you to follow our code style, but that's something we can review later.

I'll defer to @swalkinshaw on whether this is the direction we want to take with Bedrock. My personal take is that this PR is making too many changes, and we should be having a discussion about each of these different settings instead of adding all of them in a single PR.

File: .env.example
Comment (DSGND) [reply .env.example:21]: Yep, my bad, I've coded it too fast! ^^'

**Pull request closed without merging**
